status/statusgo_backend_new/contacts.nim

```diff
@@ -0,0 +1,41 @@
+import json, strmisc
+import core, utils
+import response_type
+
+export response_type
+
+proc getContacts*(): RpcResponse[JsonNode] {.raises: [Exception].} =
+  let payload = %* []
+  result = callPrivateRPC("contacts".prefix, payload)
+
+proc getContactById*(id: string): RpcResponse[JsonNode] {.raises: [Exception].} =
+  let payload = %* []
```

Comment:
Should `id` be sent in the payload?

```suggestion
  let payload = %* [id]
```

Reply:
@michaelsbradleyjr yes, you're absolutely right, most likely that is missed cause that method is just added, but not used yet.